### DISCUSSION TITLE
elixir: Clustering on Kubernetes

### DISCUSSION THREAD

**Discussion by giscus** [mbuffa/mbuffa.github.io#7] [Announcements]
# elixir: Clustering on Kubernetes

A quick walkthrough to setup a cluster with Elixir, Kubernetes and libcluster.

https://mbuffa.github.io/tips/20201022-elixir-clustering-on-kubernetes/

**Comment by ndrean** [comment]
Hi! Thanks for sharing. How would this work if instead, I run containers of releases?

**Reply by mbuffa** [reply]
Hi! Sorry, completely missed your comment.

Configuration should be pretty much the same with Mix releases, although I'd use the Kubernetes DNS strategy now instead of the Kubernetes API polling one.
